storage/innobase/lock/lock0lock.cc

```diff
@@ -60,11 +61,19 @@ my_bool innodb_deadlock_detect;
 /** The value of innodb_deadlock_report */
 ulong innodb_deadlock_report;
 
+#if defined(UNIV_DEBUG) || \
+    defined(INNODB_ENABLE_XAP_UNLOCK_UNMODIFIED_FOR_PRIMARY)
+/** Enable unmodified records unlocking on XA PREPARE for master. */
+extern my_bool innodb_enable_xap_unlock_unmodified_for_primary_debug;
+#endif /* defined(UNIV_DEBUG) ||
+          defined(INNODB_ENABLE_XAP_UNLOCK_UNMODIFIED_FOR_PRIMARY) */
```

Comment:
We could omit a subsequent occurrence of "`#if` spam" by defining an `#else` branch:
```c++
#if defined UNIV_DEBUG || defined INNODB_VERY_LONG_NAME
extern my_bool innodb_very_long_name;
#else
static constexpr bool innodb_very_long_name=false;
#endif
```
In this way, the "user" of this parameter may simply refer to it by name, without any run-time penalty.

I don’t think that it is necessary to repeat the `#if` condition in comments after `#else` or `#endif` if the enclosed code sections are short, like in this case.